csi/watch-vol-changes.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+CONFIG_MAP_INFO_DIR="/var/lib/gluster"
+
+ps aux | grep python
+
+CSI_PROCESS_ID=$(ps aux|grep python | grep main | awk '{ print $2}' | xargs);
+
+echo $CSI_PROCESS_ID;
+
+while true; do
```

Comment:
Should we trap sigup and kill that loop?

Reply:
Nope. The way I thought of it, without `-m` (or `--monitor`) option, the inotifywait breaks after 1st event. We expect this script to be running forever, and hence not planning to trap the signals here.

Reply:
Added more comments to the file.